spec/requests/api/v1/listings/create_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe 'POST api/v1/listings', type: :request do
+  let(:email)                 { 'test@test.com' }
+  let(:first_name)            { 'usu' }
+  let(:last_name)             { 'ario' }
+
+  let(:user) { create(:user, first_name: first_name, email: email, last_name: last_name) }
```

Comment:
i think there's no need to specify the first_name, last_name and email. Just use the ones created by the factory, and compare against those in your expectations